tools/AI.py

```diff
@@ -0,0 +1,9 @@
+import random;
```

Comment:
dont need the ';' at the end of import in python but not sure if it will really affect anything